packages/ses/src/enable-property-overrides.js

```diff
@@ -88,42 +87,46 @@ export default function enablePropertyOverrides(
     if ('value' in desc && desc.configurable) {
       const { value } = desc;
 
-      function getter() {
-        return value;
-      }
+      const isDebug = setHas(debugProperties, prop);
+
+      // We use concise method syntax to be `this` sensitive but still
+      // omit a prototype property of [[Construct]] behavior.
+      const { getter, setter } = {
+        getter() {
+          return value;
+        },
+        setter(newValue) {
```

Comment:
If we're going there, I'd rather go all the way and do
```suggestion
      const { get: getter, set: setter } = getOwnPropertyDescriptor({
        get [prop]() {
          return value;
        },
        set [prop](newValue) {
```

At least the `name` of each accessor would be correct. I know it'd be yet another observable change however.